src/popup.js

```diff
@@ -1,20 +1,14 @@
 import React, { Component } from "react";
 
 class Popup extends Component {
-  constructor(props) {
-    super(props);
-
-    this.handleCloseArticle = this.handleCloseArticle.bind(this);
-  }
-
-  handleCloseArticle() {
-    this.props.onCloseArticle();
-  }
```

Comment:
No need to wrap this function in another function. You can just use `this.props.onCloseArticle` directly.